### PR TITLE
Basic function

### PR DESCRIPTION
**Pull request by zkddut**
-- tweet by button click

### CODE REVIEW

File: tweet_bear.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+def photoTweet():
+  tweetTextPhoto('Nora tweet from rasp', photoCapture());
+
+
+def photoCapture():
+  filename = "{0:%Y}-{0:%m}-{0:%d}-{0:%H}-{0:%M}-{0:%S}".format(datetime.now())
+
+  camera = PiCamera()
+  camera.start_preview(alpha=190)
```
Comment (YiyingW) [inline tweet_bear.py:14]: what does this function do?

File: tweet_bear.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+from picamera import PiCamera
+from datetime import datetime
+from gpiozero import Button, LED
+from time import sleep
+from tweeter import *
+
+def photoTweet():
+  tweetTextPhoto('Nora tweet from rasp', photoCapture());
```
Comment (YiyingW) [inline tweet_bear.py]: remove nora.

File: tweet_bear.py (hunk 1 above)
Comment (zkddut) [reply tweet_bear.py:14]: How translucent the preview will be.

File: tweet_bear.py (hunk 2 above)
Comment (zkddut) [reply tweet_bear.py]: OK.

**Review by YiyingW — APPROVED** [review]

**Pull request merged**